src/porepy/applications/verification_setups/manu_poromech_nofrac.py

```diff
@@ -716,52 +709,49 @@ def body_force(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         return external_sources
 
 
-class ModifiedEquationsPoromechanics(ModifiedMassBalance, ModifiedMomentumBalance):
+class ManuPoroMechEquationsPoromechanics(
```

Comment:
The name seems unneccessarily long. Drop "Poromechanics"?

Reply:
Done.